src/EnergyPlus/Coils/CoilCoolingDXCurveFitPerformance.cc

```diff
@@ -351,6 +366,9 @@ void CoilCoolingDXCurveFitPerformance::simulate(EnergyPlus::EnergyPlusData &stat
     // calculate crankcase heater operation
     if (state.dataEnvrn->OutDryBulbTemp < this->maxOutdoorDrybulbForBasin) {
         this->crankcaseHeaterPower = this->crankcaseHeaterCap;
+        if (this->crankcaseHeaterCapacityCurveIndex > 0) {
+            this->crankcaseHeaterPower = Curve::CurveValue(state, this->crankcaseHeaterCapacityCurveIndex, state.dataEnvrn->OutDryBulbTemp);
+        }
```

Comment:
This should be `*=` to multiply by the curve value.

Reply:
Oh, I see, this is not a normalized curve.

    Curve:Linear,
      heaterCapCurve,          !- Name
      60.0,                    !- Coefficient1 Constant
      -2.,                     !- Coefficient2 x
      -30,                     !- Minimum Value of x
      40;                      !- Maximum Value of x

Reply:
Use:

    Curve:Linear,
      heaterCapCurve,
      0.428571428571429,
      0.0142857142857143,
      -30,
      40;

![image](https://github.com/NREL/EnergyPlus/assets/5778156/182c6dcc-bb93-4dcf-ad76-29cc93c8dc88)

Reply:
I'll change it to a normalized curve and multiply that by the crankcase heater capacity. @rraustad

Reply:
@rraustad Forgot to ping you. I have updated the curve to be multiplied by the crankcase heater capacity. Would you mind taking another look? Thanks :)

Reply:
Yes, this looks better.